cts-java/src/test/java/org/ga4gh/cts/api/TestData.java

```diff
@@ -30,7 +31,7 @@ private TestData() {
      * The default ID of the dataset that holds the test data.  We use something readable so the
      * meaning is clear, but in reality the value of this is unlikely to be human-readable.
      */
-    public static final String DEFAULT_DATASET_ID = "compliance-dataset";
+    public static final String DEFAULT_DATASET_ID = "WyJicmNhMSJd";
```

Comment:
The compliance data set is fixed for now. We have discussed adding another type of tests that discovers the data sets as it goes. Until we implement something like that I think we should leave the hard-coding in place.